src/imageLoader/wadors/metaData/metaDataProvider.js

```diff
@@ -33,6 +73,8 @@ function metaDataProvider(type, imageId) {
   }
 
   if (type === 'imagePlaneModule') {
+    //metaData = fixNMMetadata(metaData);
```

Comment:
Why is this commented out?

Reply:
I have here some Salim dicom images that only has position information (00200032, 00200037) in the Detector Information Sequence, not in the usual dicom tags. Ali Reza says that this NM files are incorrect, not conformat, so I put this function in a separeted file waiting for more this NM image problems arise, if so.